test/async-hooks/test-timerwrap.setTimeout.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const tick = require('./tick');
+const initHooks = require('./init-hooks');
+const { checkInvocations } = require('./hook-checks');
+const TIMEOUT = 1;
```

Comment:
So … this test is currently flaky, because setting a timeout of 1 ms sometimes ends up expiring during the tick itself, so it gets run before the destroy `uv_idle_t`, making the test fail. It’s a pretty reliable failure in debug mode, if you want to try it.

The easy approach would be to just set `TIMEOUT = common.platformTimeout(100)` or something like that, and coming up with something else might be tricky because this tests the timers implementation itself…